### ISSUE TITLE
is the development of this tool still on? 

### ISSUE THREAD

**Issue by salvob41** [Dariasteam/Geode#3]
This project is really interesting, I wonder if it is blocked and/or if you need a hand

**Comment by Dariasteam** [comment]
Sadly currently I don't have enough time for this. It was a project for a subject and which I already passed. I want to fix some thing and migrate to godot 3.1 when I can, though.

**Comment by salvob41** [comment]
community could help :D

**Comment by Dariasteam** [comment]
Absolutely, but I'm not sure if I could help a lot. Currently I'm  moving other projects to 3.1, gonna try to de the same with this at least.

**Comment by LinuxUserGD** [comment]
What about Godot 3.1 Stable?

**Comment by LinuxUserGD** [comment]
This would fix #2
